redaxo/src/core/lib/form/form_base.php

```diff
@@ -998,6 +1002,33 @@ public function getMessage()
         return $message;
     }
 
+    /**
+     * @return void
+     */
+    public function setFormAttribute(string $attributeName, ?string $attributeValue)
+    {
+        $attributeName = preg_replace('/[^\w\d\-]/', '', strtolower($attributeName));
+
+        if (0 === mb_strlen($attributeName)) {
+            return;
+        }
+
+        if (null === $attributeValue && array_key_exists($attributeName, $this->formAttributes)) {
+            unset($this->formAttributes[$attributeName]);
+            return;
+        }
+
+        if ('id' === $attributeName) {
+            $this->setFormId($attributeValue);
+        }
```

Comment:
Hier fehlt ein Return, oder?

```suggestion
            $this->setFormId($attributeValue);
            return;
        }
```